documentation/modules/oauth/ref-example-permissions-for-kafka-operations.adoc

```diff
@@ -0,0 +1,183 @@
+[id="ref-example-permissions-for-kafka-operations_{context}"]
+= Example permissions required for Kafka operations
+
+[role="_abstract"]
+
+The following examples demonstrate the user permissions required for performing common operations on Kafka.
+
+.Creating a topic
```

Comment:
Gerunds and reference? We could change these to _Topic creation_ etc if we want to avoid them

Reply:
Well, a list of examples can constitute a reference. I changed the verbs to create, produce, etc.